include/mfmg/dealii/lanczos.hpp

```diff
@@ -35,7 +35,8 @@ class Lanczos
 
   // Operations
   std::tuple<std::vector<double>, std::vector<VectorType>>
-  solve(boost::property_tree::ptree const &params) const;
+  solve(boost::property_tree::ptree const &params,
+        VectorType initial_guess) const;
```

Comment:
I think initial guess should be optional.

Reply:
I thought about it but I don't see when we would want to use it except in the tests.